src/gui/resources/GazeboDrawer.qml

```diff
@@ -165,6 +172,34 @@ Rectangle {
     }
   }
 
+  /**
+   * About dialog
+   */
+  Dialog {
+    id: aboutDialog
+    title: "Ignition Gazebo"
+
+    modal: true
+    focus: true
+    parent: ApplicationWindow.overlay
+    width: parent.width / 4
```

Comment:
How about we make this `/ 3` and give it a maximum width so it can't get too large? It looks like this for me with a smaller window:

![image](https://user-images.githubusercontent.com/5751272/106939007-8aa37480-66d4-11eb-8d2c-4c717fdeda78.png)

Reply:
`width: parent.width / 3 > 500 ? 500 : parent.width / 3`

Something like this?

Reply:
Yes, that looks good to me, thanks!